src/unity/python/turicreate/toolkits/_data_zoo.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright © 2017 Apple Inc. All rights reserved.
+#
+# Use of this source code is governed by a BSD-3-clause license that can
+# be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+from __future__ import print_function as _
+from __future__ import division as _
+from __future__ import absolute_import as _
+import os as _os
+import sys as _sys
+import requests as _requests
+import turicreate as _tc
+import hashlib as _hashlib
+from six.moves.urllib import parse as _urlparse
+from ._pre_trained_models import _download_and_checksum_files
+
+DATA_URL_ROOT = 'https://docs-assets.developer.apple.com/turicreate/data/'
+
+def _get_data_cache_dir():
```

Comment:
Almost this exact same function already exists in `toolkits/_pre_trained_models.py`. I don't think copying and pasting it then change one line is a great idea. Why not add a parameter to the existing function for the one line change, then import it and use it here?

Reply:
Done!